### DISCUSSION TITLE
No S21 on unit - just P1P2 - use external PSU?

### DISCUSSION THREAD

**Discussion by rayzjay** [revk/xESP32-Faikout#146] [Q&A]
My aircon units (FXNQ20P wall mount and FXNQ63A concealed) don't seem to have an S21 connector according to the wiring diagrams. They have an option for IR remote but the IR receiver (BRC4C82) apparently just connects to P1P2. I'm wondering if I can supply power with an external PSU, then I could pick up the P1P2 signals on the back of the wired remote.

**Comment by dzungpv** [comment]
I will not work until revk add code to support it, you should check this projects https://github.com/Arnold-n/P1P2Serial

**Reply by revk** [reply]
Indeed, P1/P2 is somewhat different.

**Reply by dzungpv** [reply]
Yes, with additional hardware, but it is still serial. But without touch to the hardware, very hard to make it work.
